### PR TITLE
Add IntesisHome climate component

### PR DESCRIPTION
**Pull request by jnimmo**
**Description:**
Adding documentation for the IntesisHome component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#8316

### CODE REVIEW

File: source/_components/climate.intesishome.markdown [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+To set it up, add the following information to your `configuration.yaml` file:
+
+```yaml
+climate:
+  - platform: intesishome
+    username: 'YOUR_USERNAME'
+    password: 'YOUR_PASSWORD'
+```
+
```
Comment (Landrash) [inline source/_components/climate.intesishome.markdown]: Definition of configuration variables are missing.

Have a look at another component for examples on how to document the variables.

Comment (jnimmo) [inline source/_components/climate.intesishome.markdown]: Thanks Landrash, hopefully that is better? Cheers

Comment (Landrash) [inline source/_components/climate.intesishome.markdown]: Looks great.

**Comment by frenck** [comment]
Closing PR since parent PR is closed as well.
Can be re-opened if all comments in the parent PR are addressed.

**Pull request closed without merging**